.github/workflows/builds.yml

```diff
@@ -51,7 +51,6 @@ jobs:
         platforms: linux/amd64,linux/arm64,linux/arm/v7
         push: ${{ github.event_name != 'pull_request' }}
         tags: |
-          tedder42/helium_hotspot_exporter:latest
-          ghcr.io/${{ github.repository_owner }}/helium_hotspot_exporter:latest
+          cloud.canister.io:5000/locohamster/loco_exporter:latest
```

Comment:
you'll need to remove these changes so I can merge.

Reply:
will do.